pkg/pillar/zedcloud/tls.go

```diff
@@ -256,6 +261,9 @@ func UpdateTLSProxyCerts(ctx *ZedCloudContext) bool {
 		}
 	}
 
+	// updating the /etc/ssl/certs for proxy certs
```

Comment:
This comment is incorrect, right? You're modifying in /usr/local/share/

Reply:
will add more, but it eventually updates the /etc/ssl/certs file

Reply:
Yeah some more comments would help.